packages/edit-site/src/components/site-editor-routes/home.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import { Editor } from '../editor';
+import SidebarNavigationScreenMain from '../sidebar-navigation-screen-main';
+import { useSyncDeprecatedEntityIntoState } from '../editor/use-resolve-edited-entity';
+
+function HomeEditor() {
+	const homePage = useSelect( ( select ) => {
+		return unlock( select( coreStore ) ).getHomePage();
+	}, [] );
+	useSyncDeprecatedEntityIntoState( {
```

Comment:
This hook is not necessary, it's only here for backwards compatibility. (persisted the entity in the edit-site state)